rmp/src/decode/mod.rs

```diff
@@ -18,6 +18,7 @@ mod uint;
 pub use self::sint::{read_nfix, read_i8, read_i16, read_i32, read_i64};
 pub use self::uint::{read_pfix, read_u8, read_u16, read_u32, read_u64};
 pub use self::dec::{read_f32, read_f64};
+#[allow(deprecated)] // While we re-export deprecated items, we don't want to trigger warnings while compiling this crate
```

Comment:
Being unsure if this is a good idea myself, I validated that any `use` of `read_str_ref` will indeed still trigger a deprecation warning.